src/app/public/src/modules/shared/http.service.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class StacheHttpService {
```

Comment:
Should this class be `abstract`?

Reply:
The config service one is not abstract, so I'm following the pattern there. @Blackbaud-SteveBrush may have more insight on that one.